### ISSUE TITLE
error calculation

### ISSUE THREAD

**Issue by varghesealex90** [SherlockLiao/pytorch-beginner#5]
In the autoencoder example, the values printed i.e.
print('epoch [{}/{}], loss:{:.4f}' .format(epoch + 1, num_epochs, loss.data[0]))
this is the error associated with the final batch of the data right ?

**Comment by L1aoXingyu** [comment]
yes, you can compute the average loss by storing all the loss.
